docs/azure_arc_jumpstart/azure_arc_sqlsrv/azure/azure_arm_template_sqlsrv_defender/_index.md

```diff
@@ -0,0 +1,269 @@
+---
+type: docs
+title: "Arc-enabled SQL servers - Microsoft Defender for SQL servers on machines"
+linkTitle: "SQL servers on machines - Defender for Cloud"
+weight: 1
+---
+
+## Deploy an Azure Virtual Machine with Windows Server with Hyper-V to support nested virtualization
+
+The following Jumpstart scenario will walk you through on how to use the provided [Azure ARM Template](https://docs.microsoft.com/azure/azure-resource-manager/templates/overview) to deploy an Azure VM installed with Windows Server, setup Hyper-V to support nested virtualization, and create guest VM with SQL Server 2019 on Hyper-V to demonstrate Defender for Cloud for SQL servers on machines and generate alerts for SQL attacks.
+
+By the end of the guide, you will have an Azure VM **ArcBox-Client** installed with Windows Server 2019 with Hyper-V and nested Windows Server VM **ArcBox-SQL** pre-configured with SQL Server 2019, projected as an Azure Arc-enabled SQL Server, then enabled SQL assessment and Microsoft Defender for SQL servers on machines.
+
+As part of this scenario deployment Defender for SQL Servers on machine plan is enabled and Log Analytics workspace created in the scenario will be assigned as default workspace for Defender for Cloud.
```

Comment:
this sentence is a bit confusing, not sure what its trying to convey

Reply:
It is just note to the user about the Defender for SQL plan is enabled as it has cost associated with it. It will enable for all machines in the subscription, even though we create one VM in this scenario. Let me rephrase little bit.

Reply:
Updated based on your comments